external/implementations/ddf/build.gradle

```diff
@@ -1,5 +1,5 @@
 group = 'org.codice.samlconf.plugins'
-description = 'Plugin implementation for DDF'
```

Comment:
❗️ The name of the artifact that gets generated from this is of the format `ddf-[version].jar`. That name could collide with DDF distributions. Some other qualifier prefix/suffix should be added to the plugin implementation modules so they don't risk collisions or confusion.

Reply:
Fixed to be `samlconf-ddf-impl` and `samlconf-keycloak-impl`.